### ISSUE TITLE
`language_ids` is not part of the published `extension.toml`

### ISSUE THREAD

**Issue by chungweileong94** [zed-industries/zed#10256]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

As part of the [`extension.toml` of Biome extension](https://github.com/biomejs/biome-zed/blob/main/extension.toml), `language_ids` is utilized in order for the Biome language server to work properly. 

It's working fine when we test it locally via as dev extension, however, the `language_ids` got removed in published version.

Steps to reproduce:
1. Install Biome extension
2. Open the installed Biome `extension.toml` from `~/Library/Application Support/Zed/extensions/installed/biome/extension.toml`

Expected:
- `language_ids` should be part of the installed `extension.toml`

Actual:
- `language_ids` is missing.

Related issue: https://github.com/biomejs/biome-zed/issues/1

### Environment

Zed: v0.130.1 (Zed Preview)
OS: macOS 14.4.1
Memory: 16 GiB
Architecture: aarch64

### If applicable, add mockups / screenshots to help explain present your vision of the feature

Original `extension.toml`, source: https://github.com/biomejs/biome-zed/blob/main/extension.toml
![image](https://github.com/zed-industries/zed/assets/15154097/b92d384b-f884-4227-975f-f3894be2fd76)

Published `extension.toml`
![image](https://github.com/zed-industries/zed/assets/15154097/dba0c409-695b-4696-b6ea-c06a325751c3)

**Comment by maxdeviant** [comment]
I think what happened here is that the Biome extension was packaged using a version of the Zed extension CLI that didn't support that `language_ids` field.

It was compiled using the Zed extension CLI built at https://github.com/zed-industries/zed/commit/764e2567557da7c6cf1beeba1462e79cf7260f98, but support for `language_ids` wasn't added until https://github.com/zed-industries/zed/commit/39cc3c07789e077bef1f8c8698a318ca1d1394c9.

Once https://github.com/biomejs/biome-zed/pull/3 is merged and a new version of the Biome extension is published it should work again.

**Comment by maxdeviant** [comment]
v0.0.2 of the Biome extension has been published.

@chungweileong94 Does this fix the problems you were seeing?

**Comment by chungweileong94** [comment]
> v0.0.2 of the Biome extension has been published.
> 
> @chungweileong94 Does this fix the problems you were seeing?

Yeah, just tested the v0.0.2 with the Zed Preview v0.131.1, works great!
